gsms.py

```diff
@@ -206,4 +334,4 @@ def main() -> None:
 
 
 if __name__ == '__main__':
-    main()
+    main()
```

Comment:
Just add an extra line at end of file. Looks good otherwise. I will kick off the workflows once this change is made.

Reply:
Whoops that wasn't intended. Added it back